docs/project_organization.md

```diff
@@ -0,0 +1,41 @@
+# Project management
```

Comment:
Should I review that? Or do we just insert it into the repo and adjust it after the next meeting where we talk about it again?

Reply:
i'd suggest to readjust it after we talked about it